app/models/appeal.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: appeals
+#
+#  id                 :bigint(8)        not null, primary key
+#  account_id         :bigint(8)        not null
+#  account_warning_id :bigint(8)        not null
+#  text               :text             default(""), not null
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+
+class Appeal < ApplicationRecord
+  belongs_to :account
+  belongs_to :account_strike, inverse_of: :appeal
```

Comment:
Shouldn't that be `account_warning`? More generally, we aren't doing anything with appeals and they are not mentioned in the PR description, I'd just remove them from the current PR.